triangle/triangle/Program.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+
+namespace triangle
+{
+    class Program
+    {
+        static void Main()
+
+        {
+
+            Console.WriteLine("Введите длину первого катета: ");
+
+            double a = Convert.ToDouble(Console.ReadLine());
+
+            Console.WriteLine("Введите длину второго катета: ");
+
+            double b = Convert.ToDouble(Console.ReadLine());
+
+            double c = Math.Sqrt(a * a + b * b);
+
+            Console.WriteLine("Гипотенуза равна " + c);
+
+            double s = 0.5 * a * b;
+
+            Console.WriteLine("Площадь треугольника равна " + s);
+
+            double p = a + b + c;
+
+            Console.WriteLine("Периметр треугольника равен " + p);
```

Comment:
Программа не должна делать то, чего от нее не требуют. В задании нужно найти площадь и периметр. Гипотенузу находить не требовалось. Конечно, она нужна для дальнейшего вычисления периметра, но **печатать ее значение на консоль не надо!**.
Еще одно замечание по стилю кода: у вас слишком много лишних локальных переменных. Если вы используете величину только один раз, подумайте, стоит ли вводить дополнительную локальную переменную, чтобы запомнить эту величину. Может, вычислить ее в том месте, где она используется. Конечно, во всем нужна мера, следует избегать слишком сложных и неудобочитаемых выражений. В такой ситуации дополнительные локальные переменные могут быть полезны.
```suggestion
            Console.WriteLine("Площадь треугольника равна " + 0.5 * a * b);
            Console.WriteLine("Периметр треугольника равен " + (a + b + Math.Sqrt(a * a + b * b)));
```